source/core/WorldBase.hpp

```diff
@@ -391,6 +418,113 @@ class WorldBase {
     return true;
   }
 
+  void SerializeAgentSet(std::ostream &os) {
+      os << ":::START agent_set\n";
+      os << agent_map.size() << '\n';
+
+      for (const auto &agent: agent_map) {
+        agent.second->Serialize(os);
+      }
+      os << ":::END agent_set\n";
+  }
+
+  /**
+   * Deserialize agents and add to world
+   * @param is istream
+   * @param world world that is being added to
```

Comment:
Looks like `world` is no longer a parameter and can be cut from the doxygen comment